src/ErrorProne.NET/ErrorProne.NET/Extensions/SyntaxNodeAnalysisContextExtensions.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Diagnostics.Contracts;
+using System.Text.RegularExpressions;
+using ErrorProne.NET.Common;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+namespace ErrorProne.NET.Extensions {
+    public static class SyntaxNodeAnalysisContextExtensions {
```

Comment:
It seems that we're using different 'curly-braces' strategy. To be consistent I would think this code needs to be reformatted.